controllers/controllers.js

```diff
@@ -8,5 +8,11 @@ exports.getTopics = (req, res, next) => {
         .catch(next)
 };
 
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params
+    retrieveArticleById(article_id).then((rows) => {
+        res.status(200).send(rows)
```

Comment:
Your responses need to be sent back in an object with a key - i this case {article:rows}